src/styles/components/_audiovideo.scss

```diff
@@ -114,6 +114,32 @@
     object-fit: cover;
   }
 
+  .onlyaudio {
+    color: $video-onlyaudio;
```

Comment:
Properties should be ordered bottom, color, display, font-size, left, pointer-events, position, right, text-align, top